src/jobs.py

```diff
@@ -293,7 +284,7 @@ def get_reminder_text(
 
 async def check_consultation_status_and_send_reminder(context: CallbackContext) -> None:
     """Sends reminder after check."""
-    if await context.job.data.is_valid():
+    if context.job.data.consultation_in_valid_time_range():
```

Comment:
У магической переменной дата(данные), содержащее некий магический объект вызывают магический метод. 
Предлагаю так переделать
```suggestion
    consultation = context.job.data
    if consultation.consultation_in_valid_time_range():
```